model/event/event_registry.go

```diff
@@ -14,14 +14,12 @@ type eventRegistry struct {
 	isSubscribable map[EventLogEntry]bool
 }
 
-var registry *eventRegistry
+var registry = &eventRegistry{
+	types:          map[string]eventDataFactory{},
+	isSubscribable: map[EventLogEntry]bool{},
+}
 
 func init() {
-	registry = &eventRegistry{
```

Comment:
why this change? this might be a merge oddity?

Reply:
I tried to use the registry in the init function of a file that comes earlier in alphabetical order (commit_queue_event.go) and got a seg fault.

Reply:
oh, I think the init() functions are being called out of order. undo this change and move the other init functions here.

Reply:
done.